cfgov/v1/models/learn_page.py

```diff
@@ -181,7 +181,7 @@ class EventPage(AbstractFilterPage):
     youtube_url = models.URLField(
         "Youtube URL",
         blank=True,
-        help_text="Format: https://www.youtube.com/embed/video_id. It can be obtained by clicking on Share > Embed on Youtube.",  # noqa: E501
+        help_text="Format: https://www.youtube.com/embed/video_id. It can be obtained by clicking on Share > Embed on Youtube.",
```

Comment:
To make the linter happy, I'd recommend something like:

```python
        help_text=("Format: https://www.youtube.com/embed/video_id. "
                   "It can be obtained by clicking on Share > Embed "
                   "on Youtube."),
```